.github/workflows/build-test.yml

```diff
@@ -37,9 +37,8 @@ jobs:
       - "check-docs"
     uses: playframework/.github/.github/workflows/cmd.yml@v3
     with:
-      java: 17, 11
-      scala: 2.12.x, 2.13.x, 3.x
-      cmd: sbt ++$MATRIX_SCALA test
+      java: 21, 17, 11
```

Comment:
I think it makes sense to just test all LTS versions, just to catch problems early with newer releases. Doesn't hurt.